content/applications/hr/time_off.rst

```diff
@@ -510,10 +529,6 @@ top-right corner of the page to view the data in that specific way. The various
 :guilabel:`List` (the default view), :guilabel:`Graph`, :guilabel:`Pivot` table, or
 :guilabel:`Calendar` view.
```

Comment:
I'd consider adding the appropriate icons here, as well... just for a little added "visual value"